docs/secure-swarm-tls.md

```diff
@@ -73,49 +73,49 @@ certificates to increase security.
 
 <!--[metadata]>Need to know about encryption too<![end-metadata]-->
 
-You can configure both the Docker Engine CLI and the Engine daemon to require
+You can configure both the Docker CLI and the Docker daemon to require

```

Comment:
@doronp You've made changes which are inconsistent with our terminology guidelines.  These are:

## Docker Engine
Docker Engine is the core product which provides image and container functionality.  
First use on a page within text, not menu titles.

* Docker Engine 
* Docker Engine daemon 
* Docker Engine CLI
* a Docker Engine client 
* Docker Engine host
* Docker Engine Remote API

Subsequent references in text

* Engine daemon
* Engine CLI
* Engine CLI commands
* an Engine command
* an Engine client
* Engine Remote API
* Engine host
* Engine